databases/week3/db_homework_week3.sql

```diff
@@ -0,0 +1,339 @@
+    `location` VARCHAR(255),
+    `when` DATETIME,
+    `max_reservations` INT(10),
+    `price` DECIMAL,
+    `created_date` DATE
+  );
+CREATE TABLE `reservation` (
+    `id` INT(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `number_of_guests` INT (10),
+    `meal_id` INT (10) UNSIGNED NOT NULL,
+    `created_date` DATE,
+    `contact_phonenumber` VARCHAR (25),
+    `contact_name` VARCHAR (255),
+    `contact_email` VARCHAR (255),
+    CONSTRAINT `fk_meal` FOREIGN KEY (meal_id) REFERENCES meal (id) ON DELETE CASCADE
+  );
+CREATE TABLE `review` (
+    `id` INT(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `title` VARCHAR (255),
+    `description` TEXT,
+    `meal_id` INT(10) UNSIGNED NOT NULL,
+    `stars` INT (10),
+    `created_date` DATE,
+    CONSTRAINT `fk_meal` FOREIGN KEY (meal_id) REFERENCES meal (id) ON DELETE CASCADE
+  );
+INSERT INTO
+  meal (
+    `title`,
+    `description`,
+    `location`,
+    `when`,
+    `max_reservations`,
+    `price`,
+    `created_date`
+  )
+values
+  (
+    'chicken',
+    'Chicken',
+    'norde fasenvej,40',
+    '2021-09-01 11:18:05',
+    '15',
+    '150.50',
+    '2021-09-01 11:18:05'
+  ),
+  (
+    'potato',
+    'potato',
+    'Konge Nytro,17',
+    '2021-12-01 15:50:05',
+    '15',
+    '100.00',
+    '2021-12-01 15:50:05'
+  ),
+  (
+    'brouny',
+    'sweet cake',
+    'Frederiksvej,11',
+    '2021-06-08 11:30:02',
+    '10',
+    '110.00',
+    '2021-06-08 11:30:02'
+  );
+INSERT INTO
+  reservation (
+    `id`,
+    `number_of_guests`,
+    `meal_id`,
+    `created_date`,
+    `contact_phonenumber`,
+    `contact_name`,
+    `contact_email`
+  )
+VALUES
+  (
+    '2',
+    1,
+    '2021-06-08',
+    '+38097-999-48-50',
+    'Olena',
+    'cool@gmail.com'
+  ),
+  (
+    '8',
+    2,
+    '2021-05-08',
+    '+38097-999-48-50',
+    'Alen',
+    'alen@gmail.com'
+  ),
+  (
+    '1',
+    3,
+    '2021-03-08',
+    '+38097-999-48-50',
+    'Nick',
+    'nick@gmail.com'
+  ),
+INSERT INTO
+  review (
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+VALUES
+  (
+    'Excellent',
+    'taste is excellent',
+    '1',
+    '5',
+    '2021-03-08'
+  ),
+  (
+    'very good',
+    'very good food',
+    '2',
+    '4',
+    '2021-03-08'
+  ),
+  ('good', 'good taste', '3', '3', '2021-03-08');
+  
+------Queries-----
+  -----meal queries-----
+  ---Get all meals---
+SELECT
+  *
+FROM
+  meal;
+---Add a new meal---
+INSERT INTO
+  meal(
+    `title`,
+    `description`,
+    `location`,
+    `when`,
+    `max_reservations`,
+    `price`,
+    `created_date`
+  )
+values(
+    'pizza',
+    'Italian dish',
+    'Rome',
+    '2021-09-01 11:18:05',
+    '10',
+    '150.00',
+    '2020-08-25 10:10:10'
+  );
+---Get a meal with any id, fx 1---
+SELECT
+  *
+FROM
+  review
+WHERE
+  id = 1;
+---Update a meal with any id, fx 1.Update any attribute fx the title or multiple attributes---
+UPDATE
+  meal
+SET
+  title = 'cheese set',
+  descriptions = '4 kind of cheese'
```

Comment:
In your table description doesn't have S, so you have to remove the S so this query can work.